cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/01/saved_search.spec.js

```diff
@@ -60,6 +59,7 @@ const runSavedSearchTests = () => {
         });
 
         cy.setDataset(config.dataset, DATASOURCE_NAME, config.datasetType);
+        cy.osd.grabIdsFromDiscoverPageUrl();
```

Comment:
before this change, we were grabbing the aliases for WORKSPACE_ID, DATASOURCE_ID, and INDEX_PATTERN_ID when creating a workspace, data source, etc.

We need these aliases to make an API call in these tests to created a saved search.

But the way cypress aliases work is that they do not persist between tests. That is why this test could not use `before` instead of `beforeEach`. But with my change, the aliases are grabbed at the beginning of each tests, and so we can successfully move to using `before/after`